packages/cms-cli/commands/hubdb/fetch.js

```diff
@@ -0,0 +1,96 @@
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { logErrorInstance } = require('@hubspot/cms-lib/errorHandlers');
+const { downloadHubDbTable } = require('@hubspot/cms-lib/hubdb');
+
+const { validatePortal } = require('../../lib/validation');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { version } = require('../../package.json');
+
+const {
+  addConfigOptions,
+  addLoggerOptions,
+  addPortalOptions,
+  setLogLevel,
+  getPortalId,
+} = require('../../lib/commonOpts');
+const { logDebugInfo } = require('../../lib/debugInfo');
+
+const FETCH_DESCRIPTION = 'fetch a HubDB table';
+
+const action = async (args, options) => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+  const portalId = getPortalId(options);
+
+  trackCommandUsage('hubdb-fetch', null, portalId);
+
+  try {
+    const { filePath } = await downloadHubDbTable(
+      portalId,
+      args.tableId,
+      args.dest
+    );
+
+    logger.log(`Downloaded HubDB table ${args.tableId} to ${filePath}`);
+  } catch (e) {
+    logErrorInstance(e);
+  }
+};
+
+const configureFetch = yargs => {
+  yargs.command({
+    command: 'fetch <tableId> <dest>',
```

Comment:
I believe `dest` is optional so this should be `[dest]`.

Reply:
Ah good call, fixed